app/assets/javascripts/components/common/AssignCell/AssignButton.jsx

```diff
@@ -306,16 +322,17 @@ const AssignButton = ({ course, role, course_id, wikidataLabels = {}, hideAssign
     return dispatch(initiateConfirm({ confirmMessage, onConfirm, warningMessage }));
   };
 
-  const assign = (e) => {
+  const assign = async (e) => {
     e.preventDefault();
-    title.split('\n').filter(Boolean).forEach((assignment_title) => {
-      const assignment = {
-        title: decodeURIComponent(assignment_title).trim(),
-        project: project,
-        language: language,
-        course_slug: course.slug,
-        role: role
-      };
+    const assignArticle = () => {
+      title.split('\n').filter(Boolean).forEach(async (assignment_title) => {
```

Comment:
why is this async?

Reply:
Actually :sweat_smile: i was using async for this line  -
`const categoryMember = await API.checkArticleInWikiCategory(title.split('\n').map(item => item.trim()).filter(Boolean));`

Reply:
Ah, missed that - the formatting makes it hard to read. There's still an extra async in `async (assignment_title)`

Also, could you fix the indentation for this?

Reply:
Okay, i will make the changes ty for the review